src/System.IdentityModel.Tokens/Utility.cs

```diff
@@ -65,5 +65,25 @@ internal static string SerializeAsSingleCommaDelimitedString(IEnumerable<string>
 
             return sb.ToString();
         }
+
+        public static bool IsHttps(string address)
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                return false;
+            }
+
+            return address.StartsWith("https");
```

Comment:
I added some more tests. "HTTPS://" will work after converting to uri and matching the uri.scheme == "https" since internally the scheme is converted to lower case when "uri" object is created. I was referring isHttps(Uri uri) as the other overload.